src/compiler/python_generator.cc

```diff
@@ -657,7 +658,7 @@ bool PrivateGenerator::PrintServiceClass(
           out->Print("options, channel_credentials,\n");
           out->Print(
               "insecure, call_credentials, compression, wait_for_ready, "
-              "timeout, metadata)\n");
```

Comment:
Please update [this file](https://github.com/grpc/grpc/blob/master/examples/python/helloworld/helloworld_pb2_grpc.py) as a visual indicator of what these changes look like. [The `Makefile`](https://github.com/grpc/grpc/blob/master/examples/python/Makefile) will work as long as you've installed a dev version of `grpcio-tools`